package/hw8/Main.java

```diff
@@ -0,0 +1,14 @@
+package hw8;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Calculator.printAddition();
```

Comment:
So, such a calculator implementation possible, but I would like to see another way for testing.
Currently, always printed same values
Pls, rewrite